app/uk/gov/hmrc/trackingconsentfrontend/assets/test/fixtures/settingsForm.html

```diff
@@ -0,0 +1,388 @@
+
+
+
+
+                    >
+
+
+
+                    <label class="govuk-label govuk-radios__label" for="campaigns">
+                        Use cookies that remember my settings on the site
+                    </label>
+
+
+
+
+
+
+                </div>
+
+
+
+
+
+
+
+
+
+
+
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="campaigns-2" name="campaigns" type="radio" value="off"
+
+
+
+
+                    >
+
+
+
+                    <label class="govuk-label govuk-radios__label" for="campaigns-2">
+                        Do not use cookies that remember my settings on the site
+                    </label>
+
+
+
+
+
+
+                </div>
+
+
+
+
+
+            </div>
+
+        </fieldset>
+
+
+
+
+    </div>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    <div class="govuk-form-group">
+
+        <fieldset class="govuk-fieldset"  aria-describedby="settings-hint" >
+
+
+            <legend class="govuk-fieldset__legend">
+
+                <h2 class="govuk-heading-m">Cookies that remember your settings</h2>
+
+            </legend>
+
+
+            <span id="settings-hint" class="govuk-hint">
+  These cookies do things like remember your preferences and the choices you make, to personalise your experience of using the site.
+</span>
+
+            <div class="govuk-radios">
+
+
+
+
+
+
+
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="settings" name="settings" type="radio" value="on"
+
+
+
+
+                    >
+
+
+
+                    <label class="govuk-label govuk-radios__label" for="settings">
+                        Use cookies that remember my settings on the site
+                    </label>
+
+
+
+
+
+
+                </div>
+
+
+
+
+
+
+
+
+
+
+
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="settings-2" name="settings" type="radio" value="off"
+
+
+
+
+                    >
+
+
+
+                    <label class="govuk-label govuk-radios__label" for="settings-2">
+                        Do not use cookies that remember my settings on the site
+                    </label>
+
+
+
+
+
+
+                </div>
+
+
+
+
+
+            </div>
+
+        </fieldset>
+
+
+
+
+    </div>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    <h2 class="govuk-heading-m">
+        Strictly necessary cookies
+    </h2>
+    <p class="govuk-body">These essential cookies do things like remember your progress through a form (for example a licence application)</p>
+    <p class="govuk-body">They always need to be on.</p>
+    <p class="govuk-body"><p class="govuk-body">
+    <a href="/help/cookies?abc=def&amp;ghi=jkl">
+        Find out more about cookies on HMRC services
+    </a>
+</p></p>
+
+
+
+
+    <button
+
+            class="govuk-button" data-module="govuk-button">
+        Save changes
+
+    </button>
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
What's going on with all the whitespace?

Reply:
It was taken from a rendered version of the form, I didn't change what was provided to the browser.